docs/advanced-topics/corpora.md

```diff
@@ -64,6 +65,6 @@ $ gsutil -m rsync \
 
 ## Corpus backups
 
-We also keep daily zipped backups of your corpora. These can be accessed from
-the `corpus_backup` column of the fuzzer statistics page. Downloading these can
-also be significantly faster than `gsutil -m rsync` on the corpus bucket.
+We keep daily zipped backups of your corpora. These can be accessed from the
+**corpus_backup** column of the fuzzer statistics page. Downloading these can
+be significantly faster than running `gsutil -m rsync` on the corpus bucket.
```

Comment:
@mbarbella-chromium @oliverchang should we recommend using `gsutil -m cp -r` instead of `gsutil -m rsync` throughout this doc? I think `rsync` doesn't make much sense and generally can be slower.

Reply:
+1.